tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -652,7 +687,11 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
           component.id = 'dashboard';  // used in `_selectedDashboardComponent`
           container.appendChild(component);
         }
+        if (dashboard.useDataSelector) {
+          container.firstElementChild.dataSelection = this._dataSelection;
```

Comment:
Hmm, I wonder if it would perhaps be better to enable the data selector and wire up the property before actually attaching the element to the container?  It seems slightly cleaner perhaps if the element is attached when it already knows the data selection it should render, versus attaching with no data selection and then having to detect the update.